### PR TITLE
Add tests with Robolectric-Instrumentation for further refactoring

### PR DESCRIPTION
**Pull request by gfx**
`SessionTest` is the test to load `session_ja.json`

FYI [Robolectric-Instrumentation](https://github.com/gfx/Robolectric-Instrumentation) is a wrapper for Robolectric to avoid a lot of Robolectric issues.

### CODE REVIEW

**Comment by konifar** [comment]
Awesome!! It's very helpful for me! :memo:

**Comment by gfx** [comment]
Wait a minute! `./gradlew check` is missing in circle.yml.

**Comment by gfx** [comment]
Done by 36767e9

**Comment by konifar** [comment]
LGTM

But I don't have enough knowledge of testing... So anybody can review?
@hotchemi @KeithYokoma @hkurokawa

File: app/src/main/java/io/github/droidkaigi/confsched/MainApplication.java [hunk 1]
```diff
@@ -52,7 +52,9 @@ public void onCreate() {
                 .appModule(new AppModule(this))
                 .build();
 
-        Fabric.with(this, new Crashlytics());
+        if (!BuildConfig.DEBUG) {
```
Comment (hotchemi) [inline app/src/main/java/io/github/droidkaigi/confsched/MainApplication.java]: Is this change needed?

**Comment by hotchemi** [comment]
Almost LGTM!

File: app/src/main/java/io/github/droidkaigi/confsched/MainApplication.java (hunk 1 above)
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/MainApplication.java]: Oh, current Deploygate apk is debug build version.
So now this should be removed.
I will do after merge.

**Comment by konifar** [comment]
LGTM
